go.mod

```diff
@@ -8,6 +8,7 @@ require (
 	github.com/onsi/ginkgo v1.13.0 // indirect
 	golang.org/x/net v0.0.0-20200904194848-62affa334b73 // indirect
 	golang.org/x/tools v0.0.0-20200904185747-39188db58858 // indirect
+	k8s.io/api v0.19.0
```

Comment:
@robscott why is a dep included in this PR?

Reply:
Removed, thanks!